src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/reporting/SessionVisitCountsDto.kt

```diff
@@ -0,0 +1,45 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.dto.reporting
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.visitscheduler.dto.sessions.SessionCapacityDto
+import uk.gov.justice.digital.hmpps.visitscheduler.dto.sessions.SessionTimeSlotDto
+import uk.gov.justice.digital.hmpps.visitscheduler.model.VisitType
+import java.time.LocalDate
+
+data class SessionVisitCountsDto(
+  @Schema(description = "Date of Report", example = "2023-09-01", required = true)
+  val reportDate: LocalDate,
```

Comment:
no annotation to enforce required fields just swagger,... same for below attributes

Reply:
Thanks! Updated Dto and code based on the comments.